components/camel-robotframework/src/main/java/org/apache/camel/component/robotframework/RobotFrameworkCamelConstants.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.robotframework;
+
+public interface RobotFrameworkCamelConstants {
+
+    public final String ROBOT_CAMEL_EXCHANGE_NAME = "exchange";
```

Comment:
It doesn't seem generic enough as name, same for the others ROBOT_VAR

Reply:
moved them into utils and left what's used to set headers and whatnot..